applications/rag/outputs.tf

```diff
@@ -17,12 +17,17 @@ output "jupyterhub_uri" {
   value       = "http://${module.jupyterhub.jupyterhub_uri}"
 }
 
+output "jupyterhub_ip_address" {
+  description = "JupyterHub gloabl IP address"
+  value       = module.jupyterhub.jupyterhub_ip_address
+}
+
 output "jupyterhub_user" {
   description = "JupyterHub user is only required for standard authentication. Ignore, in case of IAP authentication"
   value       = module.jupyterhub.jupyterhub_user
 }
 
-output "jupyter_password" {
+output "jupyterhub_password" {
```

Comment:
Can we please update the README.md file as well, if we change output names? Otherwise users won't know how to use the terraform.

Reply:
Uodated readme